web/app/scripts/src/providers/project-config.es.js

```diff
@@ -0,0 +1,22 @@
+const ProjectConfig = function () {
+    let settings = {
+        apiRoot: 'localhost:8000'
+    };
+
+    return {
+        $get: function () {
+            return {
+                getApiRoot: () => settings.apiRoot
```

Comment:
I think the pattern we used was to have a `getter` for each set of properties at least